src/Microsoft.Azure.WebJobs.ServiceBus/Listeners/ServiceBusListener.cs

```diff
@@ -35,6 +35,8 @@ public ServiceBusListener(string entityPath, ServiceBusTriggerExecutor triggerEx
             _messageProcessor = config.MessagingProvider.CreateMessageProcessor(entityPath, _serviceBusAccount.ConnectionString);
         }
 
+        internal MessageReceiver Receiver => _receiver;
```

Comment:
Done to allow the trigger binding to access the receiver, so it can add it to the binding data collection